event-handlers/src/test/java/no/sikt/nva/nvi/events/UpsertNviCandidateHandlerTest.java

```diff
@@ -52,6 +47,12 @@ public class UpsertNviCandidateHandlerTest extends LocalDynamoTest {
     NviCandidateRepository nviCandidateRepository;
     private UpsertNviCandidateHandler handler;
 
+    public static DbPublicationDate toPublicationDate(CandidateDetails.PublicationDate publicationDate) {
```

Comment:
Could be better to have this method inside PublicationDate as toDb(). Then it can be reused elsewhere

Reply:
We can move it when it's needed elsewhere :)